ch5/crud_pydantic.py

```diff
@@ -0,0 +1,51 @@
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+
+class CreateIn(BaseModel):
+    name: str
+    nickname: str
+
+
+class CreateOut(BaseModel):
+    status: str
+    id: int
+
+
+# Create a FastAPI instance
+app = FastAPI()
+
+# User database
+USER_DB = {}
+
+# Fail response
+NAME_NOT_FOUND = HTTPException(status_code=400, detail="Name not found.")
+
+
+@app.post("/users", response_model=CreateOut)
+def create_user(user: CreateIn) -> CreateOut:
+    USER_DB[user.name] = user.nickname
+    return CreateOut(name=user.name, nickname=user.nickname, status="success", id=len(USER_DB))
```

Comment:
@datawhales 석기님 name하고 nickname은 schema에 없는데 일부러 넣으신건가요?

Reply:
```suggestion
    return CreateOut(status="success", id=len(USER_DB))
```
확인해보니까 없어도 될 것 같아요. 이렇게 되어야할 것 같습니다.

Reply:
네 좋습니다! 반영하였습니다